tests/integration/s2n_tls13_handshake_tests.py

```diff
@@ -47,7 +50,25 @@ def verify_hrr_random_data(server, client):
 
     return result
 
+def key_update_recv(server, client):
```

Comment:
if it's possible, i would love to see this ran in a loop so key update executes more than once, this would be an exercise to know we can handle more than 1 key updates correctly.